pkg/hub/controllers/cluster/reconciler.go

```diff
@@ -50,14 +50,20 @@ func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reco
 	if cr.Status.ClusterHealth == nil || time.Since(cr.Status.ClusterHealth.LastUpdated.Time) > 3*time.Minute {
```

Comment:
So if in the first run the nodeIP update fails in that case, we have to wait for next 3 min to populate back to the cluster CR. can we reduce the time ?